playbooks/create-bootstrapped-image.yml

```diff
@@ -0,0 +1,161 @@
+        ansible_user: "centos"
+
+- hosts: all
+  tasks:
+    - name: Express that we'd like to not init the cluster, initialize other variables.
+      set_fact:
+        skip_init: true
+        pre_pull_images:
+          - "centos:centos7"
+          # - "centos:tools"
+          - "nfvpe/multus:deviceid"
+          - "dougbtv/centos-network"
+          # Update this image for userspace.
+          - "bmcfall/vpp-centos-userspace-cni:0.4.0"
+          - "quay.io/coreos/flannel:v0.10.0-amd64"
+          - "nfvpe/virtdp:latest"
+          - "nfvpe/kube-api-server-amd64:deviceid"
+
+- import_playbook: kube-install.yml
+
+- hosts: master
+  become: true
+  become_user: root
+  tasks:
+
+    - name: Install tmux
+      yum:
+        name: tmux
+        state: present
+
+    - name: Download tmate binary 
+      unarchive:
+        src: https://github.com/tmate-io/tmate/releases/download/2.2.1/tmate-2.2.1-static-linux-amd64.tar.gz
+        dest: /tmp/
+        remote_src: yes        
+
+    - name: Move tmate binary 
+      shell: >
+        mv /tmp/tmate-2.2.1-static-linux-amd64/tmate /usr/local/bin/tmate
+
+    - name: Gen an ssh key
+      shell: >
+        ssh-keygen -b 2048 -t rsa -f /home/centos/.ssh/id_rsa -q -N ""
+
+    - name: Clone virt-network-device-plugin locally
+      git:
+        repo: https://github.com/zshi-redhat/virt-network-device-plugin.git
+        dest: "/home/centos/virt-network-device-plugin"
+        force: yes
+
+    - name: Clone Multus locally
+      git:
+        repo: https://github.com/intel/multus-cni.git
+        dest: /home/centos/multus-cni
+        force: yes
+
+    - name: Modify Multus to use :deviceid tagged image (for SR-IOV tutorial)
+      shell: >
+        sed -i -e 's|nfvpe/multus:latest|nfvpe/multus:deviceid|' /home/centos/multus-cni/images/multus-daemonset.yml
+
+    - name: Pull kubeadm images
+      shell: >
+        kubeadm config images pull --kubernetes-version={{ kubeadm_version }}
+
+    - name: Pre-pull necessary docker images
+      shell: >
+        docker pull {{ item }}
+      with_items: "{{ pre_pull_images }}"
+
+    - name: Jimmy in the custom api server image
+      shell: >
+        docker tag k8s.gcr.io/kube-apiserver-amd64:v1.11.2 k8s.gcr.io/kube-apiserver-amd64:orig.v1.11.2;
+        docker rmi k8s.gcr.io/kube-apiserver-amd64:v1.11.2;
+        docker tag docker.io/nfvpe/kube-api-server-amd64:deviceid k8s.gcr.io/kube-apiserver-amd64:v1.11.2
+
+    - name: Re-install cloud-init goodies
+      yum:
+        name: "{{ item }}"
+        state: present
+      with_items:
+        - cloud-init
+        - cloud-utils
+        - cloud-utils-growpart
+
+    - name: Delete the cloud-init dir and re-create dir
+      file: 
+        path: /var/lib/cloud
+        state: "{{ item }}"
+      with_items:
+        - absent
+        - directory
+
+    - name: Remove hostname
+      shell: >
+        hostnamectl set-hostname ""
+
+    - name: "Power off machine"
+      shell: "sleep 2 && poweroff"
```

Comment:
This is the appropriate use of `shell` because you're using `&&` which requires a real shell.